assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -132,10 +140,23 @@ const List = styled(Area).attrs({
   }
 `;
 
-const Li = styled.li.attrs({
-  role: 'option',
-})`
+const PageSeparator = styled(ReorderableSeparator)`
+  margin-left: ${({ width }) => 3 - 10 - width}px;
+  margin-right: ${({ width }) => 3 - width}px;
```

Comment:
Is this gap intentional?
![image](https://user-images.githubusercontent.com/21036224/78131169-e6e59a00-741a-11ea-965a-25d95a45558d.png)

It creates a "dead zone" in the middle:
https://drive.google.com/open?id=1JObfk8OUollS-rOEXnYggAsqo1FU1VD0

Reply:
Fixed